webrender/src/renderer.rs

```diff
@@ -715,6 +720,15 @@ impl Renderer {
                              TextureFilter::Linear,
                              ImageData::Raw(Arc::new(mask_pixels)));
 
+        let dummy_cache_texture_id = device.create_texture_ids(1, TextureTarget::Array)[0];
+        device.init_texture(dummy_cache_texture_id,
```

Comment:
This is minor, but am I right assuming we're leaking this texture, or is there something preventing that? If not, would it make sense to destroy it on drop?